05-FAQ.md

```diff
@@ -18,24 +18,19 @@ Yes ! You can:
  * [Submit](../write) paper for the work you've already replicated
  * Spread the word about Re**Science** in your community
    ([twitter](http://twitter.com/ReScienceEds), mailing lists, blogs, etc.)
- * [Star the project](https://github.com/ReScience/ReScience) on github
```

Comment:
I didn't mean to delete this (nor the following deletion). Perhaps a mistake from editing on the Github UI. I'll see whether I can get those back in.